testsuite/integration/microprofile-tck/reactive-streams-operators/pom.xml

```diff
@@ -0,0 +1,151 @@
+
+    <properties>
+        <jbossas.ts.integ.dir>${basedir}/../..</jbossas.ts.integ.dir>
+        <jbossas.ts.dir>${jbossas.ts.integ.dir}/..</jbossas.ts.dir>
+        <jbossas.project.dir>${jbossas.ts.dir}/..</jbossas.project.dir>
+        <ts.elytron.cli>${jbossas.ts.dir}/shared/enable-elytron.cli</ts.elytron.cli>
+        <!--
+            Change how the server is provisioned since we are not part of the standard configs.
+            Since we're not shipping it in our standard configs, it makes no sense to use
+            @ServerSetup.
+            Essentially by default it will work like -Dts.layers
+        -->
+        <ts.copy-wildfly.phase>none</ts.copy-wildfly.phase>
+        <ts.microprofile-tck-provisioning.phase>compile</ts.microprofile-tck-provisioning.phase>
+        <!-- These properties control what layers are provisioned if galleon provision occurs -->
+        <ts.microprofile-tck-provisioning.base.layer>cloud-server</ts.microprofile-tck-provisioning.base.layer>
+        <ts.microprofile-tck-provisioning.decorator.layer>microprofile-reactive-streams-operators</ts.microprofile-tck-provisioning.decorator.layer>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.eclipse.microprofile.reactive-streams-operators</groupId>
+            <artifactId>microprofile-reactive-streams-operators-tck</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.microprofile.reactive-streams-operators</groupId>
+            <artifactId>microprofile-reactive-streams-operators-api</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.arquillian.testng</groupId>
+            <artifactId>arquillian-testng-container</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+    </dependencies>
+
+    <build>
+        <finalName>microprofile-reactive-streams-operators-${project.version}</finalName>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <includes>
+                        <include>org.eclipse.microprofile.reactive.streams.operators.tck.arquillian.ReactiveStreamsArquillianTck</include>
+                    </includes>
+                    <dependenciesToScan>
+                        <dependency>org.eclipse.microprofile.reactive-streams-operators:microprofile-reactive-streams-operators-tck</dependency>
+                    </dependenciesToScan>
+                    <environmentVariables>
+                        <!-- Let's loose the default timeout for TCK tests since in some cases (Windows mainly) this is not enough. -->
```

Comment:
I'm thinking about this... maybe we should also add in the comment a source from where we found this property: https://github.com/reactive-streams/reactive-streams-jvm/blob/master/tck/src/main/java/org/reactivestreams/tck/TestEnvironment.java

Reply:
Done locally. I am also increasing it to 500ms, since I saw a timeout on our CI